ui/imports/shared/views/chat/ChatContextMenuView.qml

```diff
@@ -124,39 +124,14 @@ StatusMenu {
         }
     }
 
-    //TODO uncomment when implemented
-//    StatusMenu {
-//        title: qsTr("Fetch messages")
-//        enabled: (root.chatType === Constants.chatType.oneToOne ||
-//                  root.chatType === Constants.chatType.privateGroupChat)
-//        StatusAction {
-//            text: "Last 24 hours"
-//            onTriggered: {
-//                root.fetchMoreMessages();
-//            }
-//        }
-
-//        StatusAction {
-//            text: "Last 2 days"
-//            onTriggered: {
-
-//            }
-//        }
-
-//        StatusAction {
-//            text: "Last 3 days"
-//            onTriggered: {
-
-//            }
-//        }
-
-//        StatusAction {
-//            text: "Last 7 days"
-//            onTriggered: {
-
-//            }
-//        }
-//    }
+    StatusAction {
+        objectName: "chatFetchMessagesMenuItem"
+        text: qsTr("Fetch messages")
+        icon.name: "checkmark-circle"
```

Comment:
This seems like a random icon, but whatever haha

Reply:
yeah, maybe `"download"` will work better here 🙂 

<img width="301" alt="image" src="https://github.com/status-im/status-desktop/assets/25482501/f9df3c6f-8562-4c55-9a1c-3948ac14f2bf">

Reply:
updated, thank you!